### ISSUE TITLE
Use external api commands

### ISSUE THREAD

**Issue by dhiogocorrea** [skrafft/react-native-jitsi-meet#188]
It would be nice the possibility of using external api with this wrapper.  [Jitsi meet external api](https://jitsi.github.io/handbook/docs/dev-guide/dev-guide-iframe)

**Comment by Punit-Vajpeyi** [comment]
@dhiogocorrea Did you got any solution. @skrafft Any plans of implementing this in future?

**Comment by dhiogocorrea** [comment]
@Punit-Vajpeyi No :( I'm almost trying to start implementing it

**Comment by skrafft** [comment]
Using external api was my initial plan when I started this project. I came across the following problems which led me to use the mobile frameworks provided by Jitsi instead:

- Intended for web use
- Lack of mobile support (weird message saying that mobile is not supported)
- Lack of webrtc support in Safari Webview

I don't think it has changed much lately. Are you seeking a specific feature or just ease of integration ?

**Comment by dhiogocorrea** [comment]
@skrafft I see... I'm mainly looking for the "toggleAudio" functionality. 

I was looking at this: https://github.com/jitsi/jitsi-meet/blob/master/android/sdk/src/main/java/org/jitsi/meet/sdk/ExternalAPIModule.java and wondering if creating a RNExternalAPIModule.java would work...

Thanks.

**Comment by skrafft** [comment]
I don't think it would because this module was made to exchange information between Javascript and Java, not the other way.

**Comment by dhiogocorrea** [comment]
> I don't think it would because this module was made to exchange information between Javascript and Java, not the other way.

sad...

**Comment by Punit-Vajpeyi** [comment]
@skrafft I want to listen to chat message listener since I want to notify the user when chat message arrives. Can you suggest any work around?
